### PR TITLE
Removed 'USER' instruction from Dockerfile.

### PR DESCRIPTION
**Pull request by shamlymajeed**
**Summary**:close #267

**Related issue**:#267

**Suggested reviewer(s)**:

**Reviewer checklist**:

* [ ] The headers of all files contain a reference to the repository license(i.e., "License: This file is part of go-mms")
* [ ] 100% test coverage of new code - meaning:
  * [ ] The overall test coverage increased or remained the same as before
  * [ ] Every function is accompanied with a test suite
  * [ ] Tests are both positive (testing that the function work as intended with valid data) and negative (testing that the function behaves as expected with invalid data, e.g., that correct exceptions are thrown)
  * [ ] Functions with optional arguments have separate tests for all options
* [ ] Examples are supported by doctests
* [ ] All tests are passing
* [ ] All names (e.g., files, classes, functions, variables) are explicit
* [ ] Documentation (as docstrings) is complete and understandable

The checklist is based on the S-ENDA conventions and definition of done (see [General Conventions](https://s-enda-documentation.readthedocs.io/en/latest/general_conventions.html)). The above points are not necessarily relevant to all contributions. In that case, please add a short explanation to help the reviewer.

### CODE REVIEW

**Review by johtoblan — APPROVED** [review]

**Comment by myggen** [comment]
Why remove the USER instruction ? It is kind of a policy to not run services as root

File: Dockerfile [hunk 1]
```diff
@@ -25,6 +25,5 @@ FROM debian:buster-slim
 COPY --from=build-app /build/app/mmsd /app/
 
 WORKDIR /app
```
Comment (johtoblan) [inline Dockerfile:27]:
```suggestion
WORKDIR /app
RUN  chmod nobody.nobody /workdir
USER nobody:nogroup
```
Maybe this works? @shamlymajeed  can you try

Comment (johtoblan) [reply Dockerfile:27]: RUN  chown nobody.nobody /workdir

File: Dockerfile [hunk 2]
```diff
@@ -21,10 +21,10 @@ RUN make test
 
 # SECOND STAGE: create the app runtime image.
 FROM debian:buster-slim
-
 COPY --from=build-app /build/app/mmsd /app/
-
 WORKDIR /app
+
+RUN chown nobody.nogroup /app
```
Comment (johtoblan) [inline Dockerfile:27]: nobody. or nobody:?

**Comment by myggen** [comment]
Both works

File: Dockerfile (hunk 2 above)
Comment (myggen) [reply Dockerfile:27]: Both works

**Review by johtoblan — APPROVED** [review]
LGTM

**Pull request merged**